dist/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone Project</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+<script defer src="runtime.js"></script><script defer src="main.js"></script></head>
+<body>
+    <header>
+        <div id="logo">
+            <span>Movies</span>
+            <span>&</span>
+            <span>TV Series</span>
+        </div>
+        <div id="link">
+            <a href="#">Home</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+        </div>
+    </header>
```

Comment:
Good job with customizing the concept 💯  But let's follow the layout provided. Please put the elements exactly how it looks on the design. 👍 
<img width="554" alt="Screenshot 2023-02-08 at 9 34 25 PM" src="https://user-images.githubusercontent.com/25887253/217621563-46ec78c7-4b0b-4541-a858-a950785c9832.png">
<img width="1418" alt="Screenshot 2023-02-08 at 9 34 55 PM" src="https://user-images.githubusercontent.com/25887253/217621567-0f5794bb-ef81-41a9-b2f0-2f54578fdf47.png">